pkgs/applications/misc/stylish/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "stylish";
+  version = "0.0.1";
```

Comment:
If the upstream has no tags

https://github.com/thevinter/styli.sh/tags

use `version = unstable-YYYY-MM-DD;` where the date is the date of the commit you are using at the src.rev below.

Reply:
Version edited as requested.